unit-test/data/random_useragent.json

```diff
@@ -0,0 +1,6357 @@
+{
+    "linux-samsungbrowser": [
+        {
+            "percentage": "0.008%",
+            "total": "233",
```

Comment:
this doesn't seem to be used - I'd drop it

Reply:
Yeah this is all data just for testing, I updated this file.

Reply:
oh yeah, but as discussed - lets also drop it from the real-thing™️